jwql/website/apps/jwql/templates/api_landing.html

```diff
@@ -0,0 +1,152 @@
+            </ul>
+        </p>
+        <br>
+        
+        <div id="introduction">
+            <h4>What is the JWQL API?</h4><hr>
+            <p align="justify">
+                JWQL allows users to access the data available through our programmatic Application Programming Interface (API). In short, an API
+                accepts requests for services from a system, gathers the items in the request, and returns a response to the user. A popular example used to describe the
+                functionality of an API is describing the role of a waiter in a restaurant. When you enter a restaurant and sit at your table, you are typically
+                provided a menu containing all of the drink and food options. But how do you communicate your food and drink order to
+                the kitchen staff? This is where the waiter (API) comes in, you provide the waiter with your order (request), they will communicate
+                your order to the kitchen staff (system), and bring your food and drinks when they are prepared (the response). The waiter (API) disguises
+                all of the steps that are necessary to getting your final product delivered (cooking the food, counting stock, knowing recipes, etc.) but, is the 
+                link to the user and the response they wish to obtain from a system.
+            </p>
+        </div>
+        <br>
+
+        <div id="examples">
+            <h4>Examples</h4><hr>
+            <p align="justify">
+                Here we showcase a few examples of a few of the APIs that we offer. To see a full list of supported APIs click <a href="#api_list">here</a>.
+            </p><br>
+            <h5>Obtaining Filenames Associated with a Rootname</h5>
+            <p align="justify">
+                A user is interested in all of the filenames associated with a specific JWST rootname.
+            </p>
+            <pre>
+                <code class="python">
+                    from urllib import request
+                    import json
+
+                    # Build the URL https://dljwql.stsci.edu/api/roontname/service/
+                    url = 'https://dljwql.stsci.edu/api/jw93025001001_02102_00001_nrca2/filenames/' 
+                    r = request.urlopen(url) 
+                    filenames = json.loads(r.read().decode())
+
+                    print(filenames)
+
+                    {'filenames': ['jw93025001001_02102_00001_nrca2_calints.fits',
+                                    'jw93025001001_02102_00001_nrca2_rate.fits',
+                                    'jw93025001001_02102_00001_nrca2_rateints.fits',
+                                    'jw93025001001_02102_00001_nrca2_trapsfilled.fits',
+                                    'jw93025001001_02102_00001_nrca2_uncal.fits']}
+                    </code>
+             </pre>
+
+                <h5>Obtaining Filenames Associated with Proposal</h5>
```

Comment:
Ooops, some bad copy/pasting? Looks like this should be titled "Obtaining Thumbnails Associated with Instrument" or something like that.

Reply:
Yeah, looks like it. I'll get after it!